### ISSUE TITLE
Where we can place skip: true is misleading.

### ISSUE THREAD

**Issue by ep-linden** [gdubicki/gitlabform#342]
**Describe the bug**

The config file mentioned when you set `skip: true`, a "given config section is not set AT ALL for given project." 
I placed `skip: true` under one of the branches in the `branches` section because I didn't want to inherit the branch
settings from the group level. This is similar to the [inheritance break issue](https://github.com/gdubicki/gitlabform/issues/326). 

I expected `skip: true` to work in any section or at least similar to the `files` section.

**GitLabForm version**

`GitLabForm version: 2.10.0`

**GitLab version**

F.e. 14.5.2-ee

**Comment by gdubicki** [comment]
As of now „skip: true” works only for the *whole* sections, so f.e. all branches or all files.

We would like to make this more configurable in gitlabform v3, see (link).

**Comment by gdubicki** [comment]
Hmm.. With the `inherit: false` working at any place in the config it could be used as a replacement for `skip: true`. Am I right or am I missing something, @ep-linden? 🤔

**Comment by ep-linden** [comment]
Yeah, it should make the configs under the `main` branch empty because it won't inherit anything. So I would say they are similar.

**Comment by amimas** [comment]
Maybe `skip` config can be deprecated in future minor release and remove in v3. Would be nice if the tool can print warning if this flag gets used while deprecated.

**Comment by gdubicki** [comment]
@amimas : yes, we strive for exactly such way of deprecating syntax - warning first, remove support later.
